app/views/questions/_answer.html.erb

```diff
@@ -43,8 +43,17 @@
           <a href="/answers/accept/<%= answer.id %>" data-remote="true" id="answer-<%= answer.id %>-accept" class="answer-accept btn btn-sm btn-default">Accept this answer</a>  
         <% end %>
       <% end %>
+    <% if answer.likers.length != 0 %>
+        <div class="btn btn-default btn-sm" rel="popover" data-placement="left" data-html="true" data-title="Users who like this" data-content="
+            <% answer.likers.each do |user| %>
```

Comment:
I might be overstepping myself over here, but what if a lot of users like the same post? We shouldn't really name each of the 100+ people who liked it but perhaps we could always show the first 5-10 people and a `+100`-like link which triggers a modal which shows the entirety of people who liked it. 

@jywarren What do you say?